src/frontend/components/Felleskomponenter/Dropdowns/LandDropdown.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactNode } from 'react';
+
+import { Alpha3Code, getAlpha3Codes, getName } from 'i18n-iso-countries';
+import { useIntl } from 'react-intl';
+
+import { Felt, ISkjema } from '@navikt/familie-skjema';
+import { useSprakContext } from '@navikt/familie-sprakvelger';
+
+import { SkjemaFeltTyper } from '../../../typer/skjema';
+import StyledDropdown from './StyledDropdown';
+
+interface LandDropdownProps {
+    felt: Felt<Alpha3Code | ''>;
+    skjema: ISkjema<SkjemaFeltTyper, string>;
+    label?: ReactNode;
+    dynamisk?: boolean;
+}
+
+export const LandDropdown: React.FC<LandDropdownProps> = props => {
+    const intl = useIntl();
+    const [valgtLocale] = useSprakContext();
+
+    const landkoderSortertPåNavn = Object.keys(getAlpha3Codes()).sort((a, b) => {
+        return getName(a, valgtLocale) >= getName(b, valgtLocale) ? 1 : -1;
+    });
+
+    return (
+        <StyledDropdown<Alpha3Code | ''>
```

Comment:
Denne syntaksen har jeg ikke sett før! Fancy 😄

Reply:
Kjekk den ja, for å spesifisere hva slags type det skal være :)